src/VersionsJSONUtil.jl

```diff
@@ -150,32 +179,47 @@ function main(out_path)
             # Test to see if there is an asc signature:
             asc_signature = nothing
             if !isa(platform, MacOS) && !isa(platform, Windows)
+                asc_url = string(url, ".asc")
+                print(stdout, "    Downloading $(basename(asc_url))")
                 try
-                    asc_url = string(url, ".asc")
-                    @info("Downloading $(basename(asc_url))")
                     asc_filepath = WebCacheUtilities.download_to_cache(basename(asc_url), asc_url)
                     asc_signature = String(read(asc_filepath))
-                catch e
-                    if isa(e, InterruptException)
-                        rethrow(e)
+                    println(stdout, " ✓")
+                catch ex
+                    if isa(ex, InterruptException)
+                        rethrow(ex)
                     end
+                    println(stdout, " ✗")
                 end
+
             end
 
             # Build up metadata about this file
-            kind = "archive"
-            if endswith(filename, ".exe")
+            if endswith(filename, ".dmg")
+                kind = "archive"
+                extension = "dmg"
+            elseif endswith(filename, ".exe")
                 kind = "installer"
+                extension = "exe"
+            elseif endswith(filename, ".tar.gz")
+                kind = "archive"
+                extension = "tar.gz"
+            elseif endswith(filename, ".zip")
+                kind = "archive"
+                extension = "zip"
+            else
+                error("Unsupported file extension in filename: $(filename)")
             end
             file_dict = Dict(
-                "triplet" => triplet(platform),
-                "os" => meta_os(platform),
                 "arch" => string(arch(platform)),
-                "version" => string(version),
+                "extension" => extension,
+                "kind" => kind,
+                "os" => meta_os(platform),
                 "sha256" => tarball_hash,
                 "size" => filesize(filepath),
-                "kind" => kind,
+                "triplet" => triplet(platform),
                 "url" => url,
+                "version" => string(version),
```

Comment:
The reordering adds noise to the commit, could you perhaps move it to its own commit?

Reply:
I've removed the reordering.